ar_detection.py

```diff
@@ -1,21 +1,23 @@
-"""This module will threshold the IVT Dataset and identify AR candidates. Each candidate is tested to determine if it meets the various criteria for AR classification."""
+"""This module will threshold the IVT Dataset and identify AR candidates. Each candidate is tested to determine if it meets the various criteria for AR classification. Candidate areas are converted to polygons and exported as a shapefile."""
 import math
 
 import xarray as xr
 import numpy as np
 import pyproj
 import geopandas as gpd
 import pandas as pd
+import shapely
 from tqdm import tqdm
 from scipy.ndimage import label, generate_binary_structure
 from skimage.measure import regionprops
 from scipy.stats import circmean
 from haversine import haversine
 from shapely.geometry import Polygon, shape
```

Comment:
```suggestion
from shapely.geometry import shape
```
VS Code is telling me we are no longer referencing `Polygon` in this script, so I think it is now an unused import that can be removed